src/services/webclient.ts

```diff
@@ -1812,6 +1813,33 @@ export class WebClientService {
         return promise;
     }
 
+    /*
+     * Modify a conversation.
+     */
+    public modifyConversation(conversation: threema.Conversation, isPinned?: boolean): Promise<null> {
+        const DATA_STARRED = 'isStarred';
+
+        // Prepare payload data
+        const args = {
+            [WebClientService.ARGUMENT_RECEIVER_TYPE]: conversation.type,
+            [WebClientService.ARGUMENT_RECEIVER_ID]: conversation.id,
+        };
+        const data = {};
```

Comment:
`data` can be an empty field now. I guess that is okay but also a bit of unnecessary overhead.

Reply:
You mean `undefined`. )Please don't use `null` any more. I've carefully removed all occurrences of `null`. :) )

Reply:
I never said anything about `null` :slightly_smiling_face:

Reply:
What was `None` about then? Nevermind. :man_shrugging:

Reply:
Since this is not Python, "None" was meant in an English language kind of way, sorry if that was unclear. In the specification it's not rendered in monospace, maybe that's what confused you :slightly_smiling_face: To actually specify a null value, I'd use the msgpack `nil` type.